docs/querying/operators.md

```diff
@@ -187,6 +187,7 @@ vector of fewer elements with aggregated values:
 * `min` (select minimum over dimensions)
 * `max` (select maximum over dimensions)
 * `avg` (calculate the average over dimensions)
+* `group` (returns 1 over dimensions)
```

Comment:
@beorn7

Reply:
Sorry, missed that. Perhaps it's a bit dense? Even I needed to think a bit about what "returns 1 over dimensions" means.

Perhaps "(all values in the resulting vector are 1)"?

Reply:
Adapted